crates/ingress-kafka/src/consumer_task.rs

```diff
@@ -241,13 +241,6 @@ impl ConsumerTask {
         let topics: Vec<&str> = self.topics.iter().map(|x| &**x).collect();
         consumer.subscribe(&topics)?;
 
-        debug!(
-            restate.subscription.id = %self.sender.subscription.id(),
-            messaging.consumer.group.name = consumer_group_id,
-            "Assigned topic/partitions/offset: {:?}",
-            consumer.assignment()?
-        );
-
```

Comment:
This log statement is pointless because `subscribe` won't do anything until you recv from the stream the first time.